app/src/androidTest/java/net/skyscanner/backpack/compose/modal/BpkModalTest.kt

```diff
@@ -0,0 +1,71 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.modal
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.test.isPopup
+import androidx.compose.ui.test.junit4.createComposeRule
+import net.skyscanner.backpack.BpkTestVariant
+import net.skyscanner.backpack.Variants
+import net.skyscanner.backpack.compose.BpkSnapshotTest
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.demo.compose.ModalStory
+import net.skyscanner.backpack.demo.compose.ModalWithBackIcon
+import net.skyscanner.backpack.demo.compose.ModalWithoutAction
+import net.skyscanner.backpack.demo.compose.ModalWithoutActionAndTitle
+import org.junit.Rule
+import org.junit.Test
+
+class BpkModalTest : BpkSnapshotTest() {
+
+    @get:Rule
+    val rule = createComposeRule()
+
+    @Test
+    @Variants(BpkTestVariant.Default, BpkTestVariant.DarkMode, BpkTestVariant.Rtl)
```

Comment:
nit: this is done automatically without specifying - you only need to specify these if you want to record for fewer variants